minitests/iostandard/features/Makefile

```diff
@@ -0,0 +1,76 @@
+PART=xc7a50tfgg484-1
```

Comment:
Should probably be `PART?=` to enable command line overrides?

Why is `PART` different than `VIVADO_PART`?  Maybe add a comment?
Should VIVADO_PART simply be `VIVADO_PART?=$(PART)`?

Reply:
Ok, I've made those as parameters and made the VIVADO_PART equal to PART. I can't recall where but I've seen that in some cases the XRay and Vivado parts to have different suffixes.

Reply:
Did you forget to push?  I still see the old version?